lib/diagnostic.rb

```diff
@@ -29,28 +29,29 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = '#{'Jar Jar Binks'} is really a Sith Lord.'
 
```

Comment:
You want to make sure you put the variable name in the `#{}` to be interpolated. Aka `#{character}`